### PR TITLE
Update aplications.ts

### PR DESCRIPTION
**Pull request by Llyfrs**
There was variant missing "accepted" here is how the whole request looked like:

```json
{
  "applications": {
    "1329008": {
      "userID": 2088902,
      "name": "Wevie",
      "level": 72,
      "stats": {
        "intelligence": 57393,
        "endurance": 71151,
        "manual_labor": 47731
      },
      "message": "",
      "expires": 1691801480,
      "status": "accepted"
    }
  }
}
```

### CODE REVIEW

**Comment by Llyfrs** [comment]
I will try to make it it's own branch next time so all the commits aren't visible (only the relevant one) lol, I'm still learning how to do this.

File: src/api-schema/shared/applications.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { fromStructure, Schema, Structure, StructureEnum } from "@/api-schema/sc
 export const statusEnum: StructureEnum<string> = {
     id: "status",
     name: "Status",
-    values: ["active", "declined", "withdrawn"],
+    values: ["active", "declined", "withdrawn", "accepted"],
     type: String,
     incomplete: { missing: "Missing values for expired and accepted applications." },
```
Comment (DeKleineKobini) [inline src/api-schema/shared/applications.ts:9]: Could you remove accepted from the incomplete section?

Comment (Llyfrs) [reply src/api-schema/shared/applications.ts]: Done :+1:

**Comment by Llyfrs** [comment]
I just watched an application expire and it didn't get its own flag, it just got deleted, so it isn't missing and there are only the four that are written in this request.

**Pull request merged**